eslint.config.mjs

```diff
@@ -21,8 +22,18 @@ export default [
       },
     },
     rules: {
+      'no-console': 'warn',
```

Comment:
Er zitten er nog een aantal in. Voor azure insights geloof ik? (omdat console.info niet mogelijk was).Komen daar uitzonderingen voor?